src/shared/components/Snackbar/Snackbar.style.ts

```diff
@@ -1,72 +1,128 @@
+  &.snackbar-exit-active {
+    transform: translateY(0) translateX(-150%);
+    height: 0;
+    margin-bottom: 0;
   }
-  .snackbar-exit-active {
-    transition: transform 800ms ${transitions.easing};
+
+  &.snackbar-enter-active {
+    transition: height ${transitions.timings.regular} ${transitions.easing},
+      margin-bottom ${transitions.timings.regular} ${transitions.easing},
+      transform ${transitions.timings.regular} ${transitions.easing} ${transitions.timings.regular};
+  }
+  &.snackbar-exit-active {
+    transition: transform ${transitions.timings.regular} ${transitions.easing},
+      height ${transitions.timings.regular} ${transitions.easing} ${transitions.timings.regular},
+      margin-bottom ${transitions.timings.regular} ${transitions.easing} ${transitions.timings.regular};
+  }
+`
+
+export const SnackbarButton = styled(Button)`
+  position: absolute;
+  top: ${sizes(3)};
+  right: ${sizes(5)};
+  &:hover {
+    background-color: ${colors.transparent};
   }
 `
 
-export const SnackbarWrapper = styled.div`
-  background-color: ${colors.gray[800]};
+export const SnackbarAction = styled(Button)`
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  padding: ${sizes(2)};
-  padding-left: ${sizes(4)};
-  max-width: 300px;
-  width: 100%;
-  position: fixed;
-  bottom: ${SNACKBAR_POSITION_MOBILE}px;
-  left: ${SNACKBAR_POSITION_MOBILE}px;
-  z-index: ${zIndex.overlay};
-  margin: 0;
-  @media screen and (min-width: ${breakpoints.small}) {
-    padding: ${sizes(3)};
-    padding-left: ${sizes(6)};
-    max-width: 400px;
-    bottom: ${SNACKBAR_POSITION}px;
-    left: ${SNACKBAR_POSITION}px;
+  padding: 0;
+  min-width: auto;
+  height: ${sizes(8)};
+  font-size: ${sizes(3)};
+  margin-right: ${sizes(10)};
+  font-size: 16px;
+  span {
+    margin-top: -4px;
+  }
+  &:hover {
+    background-color: ${colors.transparent};
   }
 `
 
-export const SnackbarButton = styled.button`
-  border: none;
-  background: none;
-  color: ${colors.gray[300]};
-  padding: ${sizes(3)};
-  cursor: pointer;
+export const SnackbarParagraph = styled.div<SnackbarWrapperProps>`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  font-size: ${typography.sizes.body2};
+  color: ${({ variant }) => (variant === 'secondary' ? colors.gray[300] : colors.blue[100])};
+  word-break: break-all;
 `
 
-export const SnackbarParagraph = styled.p`
-  margin: 0;
-  font-size: ${typography.sizes.body2};
-  color: ${colors.white};
+export const SnackbarHeader = styled.div`
   display: flex;
   align-items: center;
-  word-break: break-all;
-  svg {
-    margin-right: ${sizes(4)};
-    width: ${sizes(6)};
-    height: ${sizes(6)};
+  height: 32px;
+`
+
+export const Submessage = styled(Text)`
+  margin-top: 10px;
+  margin-bottom: 16px;
+`
+
+export const StyledIcon = styled(Icon)`
+  margin-left: 2px;
+  margin-right: ${sizes(2)};
+  width: ${sizes(6)};
+  height: ${sizes(6)};
```

Comment:
```suggestion
  width: ${sizes(5)};
  height: ${sizes(5)};
```
Icon should be smaller